packages/mdc-top-app-bar/README.md

```diff
@@ -50,81 +51,105 @@ npm install @material/top-app-bar
 
 ```html
-<header class="mdc-top-app-bar">
+<header class="mdc-top-app-bar mdc-top-app-bar--short">
   <div class="mdc-top-app-bar__row">
     <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
       <a href="#" class="material-icons mdc-top-app-bar__navigation-icon">menu</a>
       <span class="mdc-top-app-bar__title">Title</span>
     </section>
     <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end" role="toolbar">
-      <a href="#" class="material-icons mdc-top-app-bar__action-item" aria-label="Download" alt="Download">file_download</a>
-      <a href="#" class="material-icons mdc-top-app-bar__action-item" aria-label="Print this page" alt="Print this page">print</a>
       <a href="#" class="material-icons mdc-top-app-bar__action-item" aria-label="Bookmark this page" alt="Bookmark this page">bookmark</a>
     </section>
   </div>
 </header>
 ```
 
-Top app bars can be fixed at the top of the page:
+> Short top app bars should be used with no more than 1 action icon.
+
+### Short - Always Closed
+
+Short top app bars can be configured to always appear collapsed by applying the `mdc-top-app-bar--short-collapsed` before instantiating the component :
 
 ```html
-<header class="mdc-top-app-bar mdc-top-app-bar--fixed">
+<header class="mdc-top-app-bar mdc-top-app-bar--short mdc-top-app-bar--short-collapsed">
   <div class="mdc-top-app-bar__row">
     <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
       <a href="#" class="material-icons mdc-top-app-bar__navigation-icon">menu</a>
       <span class="mdc-top-app-bar__title">Title</span>
     </section>
+    <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end" role="top-app-bar">
+      <a href="#" class="material-icons mdc-top-app-bar__icon" aria-label="Bookmark this page" alt="Bookmark this page">bookmark</a>
+    </section>
   </div>
 </header>
 ```
 
-Short top app bars should only be used with one action item:
+### Fixed
+
+Fixed top app bars stay at the top of the page and elevate above the content when scrolled. 
 
 ```html
-<header class="mdc-top-app-bar mdc-top-app-bar--short">
+<header class="mdc-top-app-bar mdc-top-app-bar--fixed">
   <div class="mdc-top-app-bar__row">
     <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
       <a href="#" class="material-icons mdc-top-app-bar__navigation-icon">menu</a>
       <span class="mdc-top-app-bar__title">Title</span>
     </section>
-    <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end" role="toolbar">
-      <a href="#" class="material-icons mdc-top-app-bar__action-item" aria-label="Bookmark this page" alt="Bookmark this page">bookmark</a>
+    <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end" role="top-app-bar">
+      <a href="#" class='material-icons mdc-top-app-bar__action-item' aria-label='Download'>file_download</a>
```

Comment:
Should we have an opening section under Variants talk about action items, since we don't show them at all in the Basic Usage section?

Then we could just leave `...` here rather than repeating them every time. (It's probably worth including one action item in the short examples to reinforce that there should be only one.)